cli/src/commands/squash.rs

```diff
@@ -197,7 +318,10 @@ pub(crate) fn cmd_squash(
             }
             SquashedDescription::Combine => {
                 let abandoned_commits = &squashed.abandoned_commits;
-                if let Some(description) = try_combine_messages(abandoned_commits, &destination) {
+                // TODO: chain the let expression, once our MSRV is at least 1.88
+                let description = try_combine_messages(abandoned_commits, &destination);
+                if !insert_destination_commit && description.is_some() {
```

Comment:
Shouldn't we use the combined message in all cases?

Can you add test for that case, `--use-destination -d`, etc.?

Reply:
> Shouldn't we use the combined message in all cases?

That felt obvious when I did it, but now I can't find why we wouldn't want to do that for a new destination…

I'll remove it.